lint-staged.config.cjs

```diff
@@ -1,11 +1,27 @@
+const { ESLint } = require('eslint');
+
+const removeIgnoredFiles = async (files) => {
+  const eslint = new ESLint();
+  const isIgnored = await Promise.all(
+    files.map((file) => {
+      return eslint.isPathIgnored(file);
+    }),
+  );
+  const filteredFiles = files.filter((_, i) => !isIgnored[i]);
+  return filteredFiles.join(' ');
+};
```

Comment:
Is this because lint-staged is passed filenames directly so it wasn't ignoring files that ESLint normally ignores?

Reply:
Yeah, forgot to put the link to the reference: https://github.com/lint-staged/lint-staged?tab=readme-ov-file#eslint--7-1